metis-normalization/src/main/java/eu/europeana/normalization/dates/extraction/dateextractors/NumericPartsRangeDateExtractor.java

```diff
@@ -0,0 +1,95 @@
+package eu.europeana.normalization.dates.extraction.dateextractors;
+
+import static eu.europeana.normalization.dates.DateNormalizationExtractorMatchId.NUMERIC_ALL_VARIANTS;
+import static eu.europeana.normalization.dates.DateNormalizationExtractorMatchId.NUMERIC_ALL_VARIANTS_XX;
+import static eu.europeana.normalization.dates.DateNormalizationExtractorMatchId.NUMERIC_RANGE_ALL_VARIANTS;
+import static eu.europeana.normalization.dates.DateNormalizationExtractorMatchId.NUMERIC_RANGE_ALL_VARIANTS_XX;
+
+import eu.europeana.normalization.dates.DateNormalizationExtractorMatchId;
+import eu.europeana.normalization.dates.DateNormalizationResult;
+import eu.europeana.normalization.dates.edtf.EdtfDatePart;
+import eu.europeana.normalization.dates.edtf.InstantEdtfDate;
+import eu.europeana.normalization.dates.edtf.IntervalEdtfDate;
+import eu.europeana.normalization.dates.extraction.NumericPartsPattern;
+import eu.europeana.normalization.dates.extraction.NumericPartsPattern.NumericRangeDateDelimiters;
+import eu.europeana.normalization.dates.sanitize.DateFieldSanitizer;
+
+/**
+ * Patterns for numeric date ranges with variations in the separators of date components.
+ * <p>We reuse the already existent {@link NumericPartsDateExtractor} code for the edges.</p>
+ */
+public class NumericPartsRangeDateExtractor implements DateExtractor {
+
+  private static final NumericPartsDateExtractor NUMERIC_WITH_MISSING_PARTS_DATE_EXTRACTOR = new NumericPartsDateExtractor();
+
```

Comment:
Add dococumentation, that the first pattern that match the date is returned.

Reply:
Added.